yt_dlp/extractor/wetv.py

```diff
@@ -0,0 +1,198 @@
+import re
+import time
+import urllib.parse
+
+from yt_dlp import InfoExtractor
+from yt_dlp.aes import aes_cbc_encrypt
+from yt_dlp.compat import compat_ord, compat_str
+from yt_dlp.utils import bytes_to_intlist, random_user_agent
```

Comment:
```suggestion
from .common import InfoExtractor
from ..aes import aes_cbc_encrypt
from ..compat import compat_ord
from ..utils import bytes_to_intlist, random_user_agent
```
1. Use relative imports
2. Don't use `compat_str` but `str`